packages/desktop-client/src/components/budget/rollover/CoverTooltip.tsx

```diff
@@ -1,46 +1,18 @@
-import React, { type ComponentProps, useState } from 'react';
+import React, { useState } from 'react';
 
 import { useCategories } from '../../../hooks/useCategories';
 import { CategoryAutocomplete } from '../../autocomplete/CategoryAutocomplete';
 import { Button } from '../../common/Button';
 import { InitialFocus } from '../../common/InitialFocus';
 import { View } from '../../common/View';
-import { Tooltip } from '../../tooltips';
 import { addToBeBudgetedGroup } from '../util';
 
 type CoverTooltipProps = {
-  tooltipProps?: ComponentProps<typeof Tooltip>;
   onSubmit: (categoryId: string) => void;
   onClose: () => void;
 };
-export function CoverTooltip({
-  tooltipProps,
-  onSubmit,
-  onClose,
-}: CoverTooltipProps) {
-  const _onSubmit = (categoryId: string) => {
-    onSubmit?.(categoryId);
-    onClose?.();
-  };
 
-  return (
-    <Tooltip
-      position="bottom-right"
-      width={200}
-      style={{ padding: 10 }}
-      {...tooltipProps}
-      onClose={onClose}
-    >
-      <Cover onSubmit={_onSubmit} />
-    </Tooltip>
-  );
-}
-
-type CoverProps = {
-  onSubmit: (categoryId: string) => void;
-};
-
-function Cover({ onSubmit }: CoverProps) {
+export function CoverTooltip({ onSubmit, onClose }: CoverTooltipProps) {
```

Comment:
We should consider renaming this as they're no longer tooltips. Maybe `CoverMenu`?

Reply:
Didn't want to do too much changes in this PR, but that makes sense :) 
Updated